packages/taquito-contracts-library/src/errors.ts

```diff
@@ -1,12 +1,4 @@
-/**
- *  @category Error
- *  @description Error that indicates an invalid address being used or passed
- */
-export class InvalidAddressError extends Error {
-  constructor(message: string) {
-    super(message);
-  }
-}
+export * from '@taquito/core';
```

Comment:
this file path seem renamed to `packages/taquito-contracts-library/src/taquito-core.ts` ?